virttest/qemu_devices/qcontainer.py

```diff
@@ -1796,6 +1847,28 @@ def define_hbas(qtype, atype, bus, unit, port, qbus, pci_bus,
 
         return devices
 
+    def _retrieve_access_info(self, image, params):
```

Comment:
Seems to be duplicate with the one in `qemu_storage`

Reply:
They play different roles.
The behavior in  qemu-img and qemu-kvm is quite different, so I have to define two _retrieve_access_info.